examples/barchart-grouped.rs

```diff
@@ -202,8 +196,7 @@ impl Company {
     /// on the bar
     fn horizontal_revenue_bar(&self, revenue: u32) -> Bar {
         let text_value = format!("{} ({:.1} M)", self.name, f64::from(revenue) / 1000.);
-        Bar::default()
-            .value(u64::from(revenue))
+        Bar::new("", u64::from(revenue))
```

Comment:
`Bar::new("", /* */)` and `Bar::default()` are not the same here. One of them has `None` as the `label` and the other one is empty string. I would prefer the previous way of constructing it just to make sure we have the same behavior. (Same for the other parts where we use `""` in construction)